h/templates/home.html.jinja2

```diff
@@ -9,7 +9,7 @@
   {{ super() }}
   <link rel='canonical' href='{{ base_url }}'>
```

Comment:
It would be nice to clean up these last few references to `base_url` so we can get rid of it -- I don't think there's any reason we shouldn't just use `request.route_url('index')` here.